scripts/bootstrapper/run.ps1

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env powershell
+#requires -version 4
+
+<#
+.SYNOPSIS
+Build this repository
+
+.DESCRIPTION
+Downloads korebuild if required. Then builds the repository.
```

Comment:
"Downloads KoreBuild if required. Then executes the KoreBuild command. To see available commands, execute with \`-Command help\`"